### PR TITLE
[9.0][PORT] account_move_locking

### PR DESCRIPTION
**Pull request by vrenaville**
I have switch from period to date the account move locking module

### CODE REVIEW

**Comment by yvaucher** [comment]
You might want to add constraints to not have start_date > end_date

File: account_move_locking/wizard/account_lock_account_move.py [hunk 1]
```diff
@@ -29,34 +29,37 @@ class lock_account_move(models.TransientModel):
...
+    date_start = fields.Datetime(string='Date start',
+                                 required=True)
+    date_end = fields.Datetime(string='Date end',
+                               required=True)
 
     @api.multi
     def lock_move(self, data):
+        if self.date_start > self.date_end:
+            raise exceptions.UserError(_('Date start need to be \
+                                          before Date end'))
```
Comment (yvaucher) [inline account_move_locking/wizard/account_lock_account_move.py:42]: Have you tried with @api.constrains('date_start', 'date_end') ?

Comment (vrenaville) [inline account_move_locking/wizard/account_lock_account_move.py:42]: nope I will check it. thanks

**Comment by vrenaville** [comment]
@yvaucher Thanks for the review I have done the modification according to your comment

**Comment by yvaucher** [comment]
:+1:

**Comment by jgrandguillaume** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Please, next time don't merge without reviewing latest README template. I have fixed the README in this commit: https://github.com/OCA/account-financial-tools/commit/9835bbffdb25a6a093862d1d9f850b15fd6ebb22

**Comment by yvaucher** [comment]
@pedrobaeza I was doing it ...

**Comment by yvaucher** [comment]
Ok thanks @pedrobaeza

**Comment by yvaucher** [comment]
Note that I changed README.md template in https://github.com/OCA/account-financial-tools/commit/5fbe7fa8408df75d4c3bfe2d62bee0ea306dfa7c

**Comment by sbidoul** [comment]
@yvaucher don't worry about the addons list in README.md, it's updated nightly by the bot.

**Comment by pedrobaeza** [comment]
Yeah, I'm talking about the README.rst inside the module.

**Comment by yvaucher** [comment]
@sbidoul ok I didn't know that thanks. This is clever as it avoid conflicts on README.md between PRs

**Comment by yvaucher** [comment]
@pedrobaeza I know, errors happen sometimes, thanks for the fix.

**Pull request merged**